libafl/src/mutators/token_mutations.rs

```diff
@@ -1090,37 +1090,58 @@ impl AFLppRedQueen {
     }
 }
 
-impl<I, S> MultipleMutator<I, S> for AFLppRedQueen
+impl<I, S> Mutator<I, S> for AFLppRedQueen
+where
+    S: UsesInput + HasMetadata + HasRand + HasMaxSize + HasCorpus,
+    I: HasBytesVec + From<Vec<u8>>,
+{
+    fn mutate(
+        &mut self,
+        state: &mut S,
+        input: &mut I,
+        stage_idx: i32,
+    ) -> Result<MutationResult, Error> {
+        let mut result = self.multi_mutate(state, input, stage_idx, Some(1))?;
+        if let Some(first_input) = result.pop() {
+            *input = first_input;
+            Ok(MutationResult::Mutated)
+        } else {
+            Ok(MutationResult::Skipped)
+        }
+    }
+}
```

Comment:
New trait implementor? :p

Reply:
Ah I see, it's a required trait now. Nevermind.

Reply:
Maybe it shouldn't be(?) But I feel like it can be, so..

Reply:
Removed it